Meta/run.sh

```diff
@@ -314,8 +314,7 @@ if [ "$SERENITY_ARCH" = "aarch64" ]; then
     "
 else
     SERENITY_KERNEL_AND_INITRD="
-    -kernel Kernel/Prekernel/Prekernel
-    -initrd Kernel/Kernel
+    -kernel Kernel/Prekernel/Kernel
```

Comment:
If you effectively combined the prekernel and kernel into one image, why still put it under `Prekernel`?

Reply:
I think this happens because we generate a `Kernel.o` file in the "compiling kernel" stage, and only then we compile the `Prekernel`, bake them together (in the prekernel cmakelists) so this is the result, also can see via this snippet from CMakeLists.txt in the Prekernel folder:
```
add_custom_command(
    TARGET ${PREKERNEL_TARGET} POST_BUILD
    COMMAND ${CMAKE_OBJCOPY} -O ${ELF_OBJCOPY_TARGET} ${CMAKE_CURRENT_BINARY_DIR}/${PREKERNEL_TARGET} ${CMAKE_CURRENT_BINARY_DIR}/Kernel
    BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/Prekernel
)
```

Reply:
Done.